content/en/platform/corda/5.1/application-networks/creating/members/reregister.md

```diff
@@ -0,0 +1,214 @@
+---
+date: '2023-10-04'
+version: 'Corda 5.1'
+title: "Re-register a Member"
+menu:
+  corda51:
+    identifier: corda51-networks-members-reregister
+    parent: corda51-networks-members
+    weight: 6000
+section_menu: corda51
+---
+
+# Re-register a Member
+
+A {{< tooltip >}}member{{< /tooltip >}} may request to update its own member-provided context, for example,
+after key rotation or changes to its endpoint information. Additionally, a member who previously attempted to register but
+failed may wish to try again. The membership re-registration steps described in this section can be followed for both of these scenarios.
+
+The instructions on this page assume that you have completed the [registration]({{< relref "register.md" >}}) steps.
+
+You can learn more about configuring the registration process in the [Managing Members section]({{< relref "../../managing/registration-requests/_index.md" >}}).
+
+This section contains the following:
+1. [Inspect Member-Provided Context]({{< relref "#inspect-member-provided-context" >}})
+2. [Re-register a Member]({{< relref "#re-register-a-member-1" >}})
+
+## Inspect Member-Provided Context
+
+Currently, updates to the member-provided context are limited to custom properties (keys with the `ext.` prefix) and endpoint
+information only. Changes to other Corda platform properties are not currently supported.
+
+You may inspect member's current member-provided context either by performing a member lookup, or by looking up its latest
+registration request. For example, to look up Alice:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```bash
+curl --insecure -u admin:admin -X GET $API_URL/members/$HOLDING_ID?O=Alice
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Uri "$API_URL/membership/$HOLDING_ID?O=Alice" | ConvertTo-Json -Depth 4
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+Alternatively, to retrieve the registration request:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```bash
+export REGISTRATION_ID=<registration ID>
+curl --insecure -u admin:admin -X GET $API_URL/membership/$HOLDING_ID/$REGISTRATION_ID
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+$REGISTRATION_ID = <registration ID>
+Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Uri "$API_URL/membership/$HOLDING_ID/$REGISTRATION_ID"
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+To retrieve information about key pairs (for example, key ID required for the registration context) belonging to a
+holding identity, use:
+
+```bash
+curl --insecure -u admin:admin -X GET $API_URL/keys/$HOLDING_ID
+```
+
+## Re-register a Member
```

Comment:
Seems strange to have a sub-section named the same as a parent. Might suggest you don't need subsections at all?

Reply:
Ok, changed it to a numbered step. My only fear was that it would become very long but I guess indentation helps to make things more structured and legible.